.buildkite/scripts/steps/integration_tests_tf.sh

```diff
@@ -39,13 +39,19 @@ if [[ "${BUILDKITE_RETRY_COUNT}" -gt 0 || "${FORCE_ESS_CREATE:-false}" == "true"
 else
   # For the first run, we start the stack in the start_ess.sh step and it sets the meta-data
   echo "~~~ Receiving ESS stack metadata"
-  export ELASTICSEARCH_HOST=$(buildkite-agent meta-data get "es.host")
-  export ELASTICSEARCH_USERNAME=$(buildkite-agent meta-data get "es.username")
-  export ELASTICSEARCH_PASSWORD=$(buildkite-agent meta-data get "es.pwd")
-  export KIBANA_HOST=$(buildkite-agent meta-data get "kibana.host")
-  export KIBANA_USERNAME=$(buildkite-agent meta-data get "kibana.username")
-  export KIBANA_PASSWORD=$(buildkite-agent meta-data get "kibana.pwd")
-  export INTEGRATIONS_SERVER_HOST=$(buildkite-agent meta-data get "integrations_server.host")
+  if [[ $BUILDKITE_STEP_KEY == "integration-fips-ess" ]]; then
+    JOB_ID=$(buildkite-agent meta-data get "ess.job.fips")
```

Comment:
Can we use `BUILDKITE_JOB_ID` instead of `JOB_ID`? Or is `BUILDKITE_JOB_ID` a built-in env variable?

Reply:
yes @v1v I think that `BUILDKITE_JOB_ID` is already set by buildkite and is different per invocation, look at the env [here](https://buildkite.com/elastic/elastic-agent/builds/24230#01981a5d-cbfb-4418-b39c-a4c0a2afad5d) and [here](https://buildkite.com/elastic/elastic-agent/builds/24230#01981a5d-cbfe-4d09-93cd-1bfc290a85cf)

Reply:
I agree 100% that what he have here is far from ideal, hopefully when could re-visit that approach with the migration to oblt-cli, wdyt?